source/server/http/admin.cc

```diff
@@ -425,5 +427,27 @@ const Network::Address::Instance& AdminImpl::localAddress() {
   return *server_.localInfo().address();
 }
 
+bool AdminImpl::addHandler(const std::string& prefix, const std::string& help_text,
+                           HandlerCb callback, const bool removable) {
+  auto it = std::find_if(handlers_.begin(), handlers_.end(),
+                         [prefix](const UrlHandler& entry) { return prefix == entry.prefix_; });
+  if (it == handlers_.end()) {
+    handlers_.push_back({prefix, help_text, callback, removable});
+    return true;
+  }
+  return false;
+}
+
+bool AdminImpl::removeHandler(const std::string& prefix) {
+  auto it = std::find_if(handlers_.begin(), handlers_.end(), [prefix](const UrlHandler& entry) {
```

Comment:
you should be able to use handlers_.remove_if() here. What is compiler error if it does not work?